scripts/manage_flavours.py

```diff
@@ -57,6 +71,52 @@ async def update_flavour(flavour_data: dict, logger: logging.Logger):
             logger.info("Flavor %s update is completed", flavour_data["name"])
 
 
+async def prune_flavours(
+    flavours_data: [], logger: logging.Logger, confirmation_yes
+):
+    async with database.Session() as db:
+        flavour_names_in_config = list(map(lambda x: x["name"], flavours_data))
+        flavours_in_db = await pf_crud.list_flavours(db)
+
+        logger.info(
+            "Found %d flavours in database, %d flavours in config",
+            len(flavours_in_db),
+            len(flavours_data),
+        )
+
+        orphaned_flavours = []
+        for flavour in flavours_in_db:
+            if flavour.name not in flavour_names_in_config:
+                orphaned_flavours.append(flavour.id)
+                logger.info("\t%s (orphaned)", flavour.name)
+            else:
+                logger.info("\t%s", flavour.name)
+
+        logger.info("")
```

Comment:
empty log message

Reply:
It was intentional sparator but I'm fine with removing.

Reply:
Can you remove it please?

Reply:
Removed.